Makefile

```diff
@@ -5,7 +5,7 @@ APP_NAME = embedded-cluster
 ADMIN_CONSOLE_CHART_URL = oci://registry.replicated.com/library
 ADMIN_CONSOLE_CHART_NAME = admin-console
 ADMIN_CONSOLE_CHART_VERSION = 1.109.3
-ADMIN_CONSOLE_IMAGE_OVERRIDE =
+ADMIN_CONSOLE_IMAGE_OVERRIDE = ttl.sh/cbodonnell/kotsadm:24h
```

Comment:
is this still required?

Reply:
Good call out. Definitely not needed and was in here because the tests depend on the KOTS changes and KOTS CI was blocked yesterday. Should be unblocked now, so I'll get that released and the new version incorporated into this PR before merging.